### DISCUSSION TITLE
OpenFGA v1.4.1 release is out!

### DISCUSSION THREAD

**Discussion by adriantam** [openfga/community#237] [Announcements]
Hello folks,

We have just released [v1.4.1](https://github.com/openfga/openfga/releases/tag/v1.4.1) of the OpenFGA server.  This release includes new features such as support for cancellation and timeouts when evaluating Condition as well as tracing span information for Condition evaluation.  In addition, this release contains performance improvements by reducing goroutine overhead in ListObjects as well as other fixes.

If you have any questions/concerns with this release, then please don’t hesitate to reach out here on [Discord](https://discord.gg/8naAwJfWN6) or here.

**Comment by jon-whit** [comment]
In this `v1.4.1` release we namespaced the custom Prometheus metrics that we export. This had a side-effect of a change in the name of our metrics. Specifically, custom OpenFGA metrics are now prefixed with `openfga_`. For example, the metric `request_duration_by_query_count_ms` is now exported as `openfga_request_duration_by_query_count_ms`. We did this to align with Prometheus best-practices and to avoid potential collisions with metrics exported by other domains. If you’re using these custom metrics in your integration(s), then you may need to add the prefixes to fix any references to these metrics you have.

Cheers 🍻
